src/rules/custom-property-empty-line-before/__tests__/index.js

```diff
@@ -0,0 +1,147 @@
+import {
+  mergeTestDescriptions,
```

Comment:
I think we can close the mergeDescriptions.

Because custom properties are nested by default and we have three `except` options, both of which reduce the amount of shared tests.